src/cuga/backend/tools_env/registry/registry/api_registry_server.py

```
@@ -140,8 +140,34 @@ async def _get_or_create_registry(
+                f"Policy system is enabled but PolicyStorage.connect() failed: {e}"
+            ) from e
+    
+    reg = ApiRegistry(client=manager, policy_storage=policy_storage)
+    try:
+        await reg.start_servers()
+    except Exception:
+        # Clean up policy_storage if start_servers fails
+        if policy_storage is not None:
+            try:
+                await policy_storage.disconnect()
+            except Exception as cleanup_error:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)